plugins/calibre.koplugin/metadata.lua

```diff
@@ -221,8 +221,8 @@ end
 
 -- cleans all temp data stored for current library.
 function CalibreMetadata:clean()
-    self.books = {}
-    self.drive = {}
+    self.books = rapidjson.array({})
```

Comment:
Same could apply to these two variables. Sorry, I didn't think about it yesterday.

Both are lua representations of json objects, after decoding them. They should be lua tablets until we need to encode them as json.